service/src/main/java/uk/gov/hmcts/reform/fpl/utils/CaseDetailsHelper.java

```diff
@@ -23,4 +27,16 @@ public static void removeTemporaryFields(CaseDetails caseDetails, String... fiel
             caseDetails.getData().remove(field);
         }
     }
+
+    public static boolean isInOpenState(CaseDetails caseDetails) {
```

Comment:
just a thought, could we pass in `String` or `State`? This is as we are currently only using one attribute in `CaseDetails`

Reply:
dont really agree, class name is CaseDetailsHelper, so it implies that it operates on CaseDetials. What client want to know is if case is in openedState, not if state is opened